libheif/plugins/heif_encoder_aom.cc

```diff
@@ -258,11 +257,11 @@ static void aom_init_parameters()
   p->version = 2;
   p->name = kParam_min_q;
   p->type = heif_encoder_parameter_type_integer;
-  p->integer.default_value = 1;
+  p->integer.default_value = 0;
```

Comment:
Dirk: I confirmed with Jingning Han (who leads our libaom team) that setting min-q to 0 or 1 doesn't make a difference unless we are doing lossless encoding. So the original default value of 1 is harmless. But it's better to use libaom's default of 0.

Reply:
What does that mean exactly? Q=0 is lossless quantization, so I assume that when in normal (non-lossless) encoding mode, the encoder will never choose Q=0?

Reply:
Dirk: That's correct. In lossy encoding mode, the encoder will never choose Q=0. So setting the minimum of Q to 0 or 1 doesn't make a difference.